cdap-master/pom.xml

```diff
@@ -138,16 +138,10 @@
     <profile>
       <id>dist</id>
       <properties>
+        <etl.batch.plugin.name>etlBatch</etl.batch.plugin.name>
+        <etl.realtime.plugin.name>etlRealtime</etl.realtime.plugin.name>
```

Comment:
This is the standard way of having maven to do properties substitution. See `ProjectInfo` class in cdap-common on the usage.